app/src/main/java/ru/vssemikoz/newsfeed/adapters/NewsFeedAdapter.java

```diff
@@ -14,78 +15,35 @@
 
 import com.squareup.picasso.Picasso;
 
-import java.util.List;
-
 import ru.vssemikoz.newsfeed.R;
 import ru.vssemikoz.newsfeed.models.NewsItem;
 import ru.vssemikoz.newsfeed.storage.IconicStorage;
 import ru.vssemikoz.newsfeed.utils.TypeConverters.DateConverter;
 
-public class NewsFeedAdapter extends RecyclerView.Adapter<NewsFeedAdapter.NewsViewHolder> {
-    private List<NewsItem> newsList;
-    private Context context;
-    private onItemClickListener mListener;
+public class NewsFeedAdapter extends BaseAdapter {
```

Comment:
You're using your `BaseAdapter` incorrectly since it's a generic class and needs a type parameter. Take a look at your `onBind` method - it accepts an `Object` as a parameter while it's wrong.